app/server/mergedServerMain.ts

```diff
@@ -144,12 +144,12 @@ export async function main(port: number, serverTypes: ServerType[],
         server.addDocApiForwarder();
       }
       server.addJsonSupport();
+      await server.addTelemetry();
```

Comment:
This is how I stupidly fix the issue with `/support-grist`. I let this method preempt `addLandingPage` (which calls `addAppEndpoint` with the changes above).

However I feel uncomfortable, as I wonder if there could not be other pages which would also be impacted.

Would that make any sense to move that at the end of the condition branch (i.e. after the call to `server.addInstallEndpoints()`)?

Reply:
I think back when the doc landing pages were added, we were careful to keep landing page lengths less than 12 to avoid this kind of potential collision. I'd be ok treating exceptions that have crept in as bugs to be fixed. @georgegevoian what do you think about renaming `/support-grist` to `/support/grist` or similar?

Reply:
> I think back when the doc landing pages were added, we were careful to keep landing page lengths less than 12 to avoid this kind of potential collision. I'd be ok treating exceptions that have crept in as bugs to be fixed. @georgegevoian what do you think about renaming `/support-grist` to `/support/grist` or similar?

Maybe just `/support` for now? We could switch to `/support/foo` once we have multiple sub-pages.

Reply:
Sounds ok to me. The meaning of "support" is a bit ambiguous, but that might be useful if we grow a management panel or similar. @fflorent are you comfortable making this change or should we do it?